src/components/input/RadioGroup.tsx

```diff
@@ -66,13 +65,20 @@ function Radio<T extends RadioValue>({
     >
       <div className="flex items-center gap-x-1.5">
         {isSelected ? <RadioCheckedIcon /> : <RadioIcon />}
-        {children}
+        <span
+          className={classnames('flex items-center gap-x-1.5', {
+            'text-grey-7 group-hover:text-grey-8 group-aria-checked:text-grey-8':
```

Comment:
For the title we set grey-8 when checked or hovered, and grey-7 otherwise, while for the subtitle we set grey-7 when checked or hovered, and grey-6 otherwise.

This ensures the title is always slightly darker than the subtitle.